core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -295,7 +298,11 @@ public static void createFile(String filePath) throws IOException {
    * @param path path of the directory to create
    */
   public static void createDir(String path) throws IOException {
-    new File(path).mkdirs();
+    try {
+      Files.createDirectories(Paths.get(path));
+    } catch (AccessDeniedException e) {
+      LOG.info("Fail to create folder " + path + "caused by AccessDeniedException");
```

Comment:
1. trick to use `LOG`: add exception as the last argument and it will be logged as the cause automatically. like this:
```java
LOG.info("Fail to create folder " + path, e);
```
2. actually i think we should throw this exception rather than swallow it. Why do you catch and convert the exception to log message?